ha/util/conf_store.py

```diff
@@ -103,6 +143,8 @@ def set_cluster_cardinality(self, index):
         # Combine the lists data_pods, server_pod, control_pods and find unique machine ids
         watch_pods = data_pods + server_pods + control_pods
         watch_pods = list(set(watch_pods))
+        matched_node_labeled_dict = ConftStoreSearch.get_matched_labels(index, watch_pods)
```

Comment:
If pods are going to have FQDN names only then  watch_pods will be empty right  ? 
Or is it the case that  machine-id will be present in the same path regardless of whether a pod is going to have FQDN or machine-id as the unique identifier ?

Reply:
machine-id will remain at the same field , nothing will be changed there.
only thing is , whatever is there in cortx.io/machine-id it wnt be the machine-id anymore as provisioner will be generating it.

so inorder to match cortx.io/machine-id from gconf, we need to match it with node><id>>node_id.

Reply:
>>so inorder to match cortx.io/machine-id from gconf, we need to match it with node>>node_id.
@tanujashinde0405 this is when the event is generated by K8S right ?

Reply:
Also the location /path of actual machine-id was going to be changed in your PR right @tanujashinde0405 ?

Reply:
actual machine-id path won't change, it will always be the same.
I had mentioned that the new id which is getting matched to cortx.io/machine-id that will be available at new field called node_id